features/every.feature

```diff
@@ -7,9 +7,9 @@ Feature:  every
   Scenario: Using cron shortcuts ':minute'
     Given a rule
       """
-      rule 'Log the rule name every minute' do
+      log_rule = rule 'Log the rule name every minute' do
```

Comment:
Could it be possible to pass the rule as a parameter to the rule block instead? Or would that not be available in the run-block's scope anyway?

```ruby
rule 'Log the rule name every minute' do |r|
  every :minute
  run { logger.info "Rule '#{r.name}' executed" }
end
```

Reply:
That would in fact be better and much more ruby like..  right now after I look at this it wouldn't work with 'on start'.  Let me see if this works.

Reply:
Updated..  
I am really glad we do code reviews.  this is much better.